### DISCUSSION TITLE
There is no "sbin" dir in the file "nginx-1.26.2.tar.gz"

### DISCUSSION THREAD

**Discussion by yuf19-2022** [nginx/nginx#371] [Q&A] [answered]
How can I do?

**Comment by HanadaLee** [comment, marked as answer]
how to build nginx
https://nginx.org/en/docs/configure.html
or
https://github.com/nginx/nginx?tab=readme-ov-file#building-from-source

Please note that if there is no special need, it is recommended to install nginx using the pre-built package

https://nginx.org/en/docs/install.html
or
https://github.com/nginx/nginx?tab=readme-ov-file#downloading-and-installing
